patches/minecraft/net/minecraft/advancements/FunctionManager.java.patch

```diff
@@ -0,0 +1,10 @@
+--- ../src-base/minecraft/net/minecraft/advancements/FunctionManager.java
++++ ../src-work/minecraft/net/minecraft/advancements/FunctionManager.java
+@@ -160,6 +160,7 @@
+ 
+     private void func_193061_h()
+     {
++        net.minecraftforge.common.ForgeHooks.loadFunctions(this.field_193070_d);
```

Comment:
What exactly are you wanting to do with functions? IIRC they are for map makers, you can do anything you want in your mods code you don't need functions.

Reply:
My goal here is to allow people to distribute advancements, functions, and structures in something much like the 'datapack' that's coming in 1.13, but loaded from outside the world files with some other things included. So I can't do what I want in my mod's code, since the mod's purpose is letting others provide their own functions.